djmoney/settings.py

```diff
@@ -1,15 +1,14 @@
 import operator
-import warnings
 from types import MappingProxyType
 
 from django.conf import settings
 
-from moneyed import CURRENCIES, DEFAULT_CURRENCY, DEFAULT_CURRENCY_CODE
+from moneyed import CURRENCIES, Currency
 
 
 # The default currency, you can define this in your project's settings module
 # This has to be a currency object imported from moneyed
-DEFAULT_CURRENCY = getattr(settings, "DEFAULT_CURRENCY", DEFAULT_CURRENCY)
+DEFAULT_CURRENCY: Currency = getattr(settings, "DEFAULT_CURRENCY", None)
```

Comment:
Should we instantiate `Currency` here?

```suggestion
DEFAULT_CURRENCY = Currency(getattr(settings, "DEFAULT_CURRENCY", None))
```

(Not sure if `Currency` takes a `Currency` so might need to be more involved)

Reply:
Nice catch 👍 , just tried passing a `Currency` instance. Didn't work. `DEFAULT_CURRENCY` has to be a `str`.

Reply:
Although, think we need to handle `None` here as well.

Reply:
Just realised the comment says "This has to be a currency object imported from moneyed". So, perhaps we shouldn't do anything new here?